cytoscape-panzoom.js

```diff
@@ -191,10 +191,14 @@ SOFTWARE.
           // functions for calculating panning
           ////////////////////////////////////
 
+          function isMouseEvent(e) {
+            return e.type.substring(0, 5) === 'mouse';
```

Comment:
Think it might be cleaner to just use 'startsWith', but I could be missing something.

Reply:
I wasn't really sure how to check whether an event is a mouse even or a touch event, so I just went with this until I'm aware of a more appropriate way.

Reply:
Yeah, unfortunately I think checking the type field is the only way.